content/challenges/easy/3 - Movies/func.js

```diff
@@ -17,5 +17,13 @@
  * @returns {boolean}
  */
 module.exports = (customerMoney, isMovieFull, rating, age, isParentWith) => {
-    // Your code here
-};
+   let saleAllowed = true;
+   if (customerMoney < 8) {
+    saleAllowed = false;
+   } if (isMovieFull === true) {
+    saleAllowed = false;
+   } if ((rating === "PG" || rating === "R") && (!isParentWith && age < 17)) {
+    saleAllowed = false;
+   }
+   return saleAllowed;
```

Comment:
This is nice and verbose. Love it. Some developers prefer return statements to exit as quickly as possible but in this case I like this more.

My ONLY critique is that when you have code pieces that are distinct don't be afraid of a line break.

```suggestion
   let saleAllowed = true;
   
   if (customerMoney < 8) {
    saleAllowed = false;
   } if (isMovieFull === true) {
    saleAllowed = false;
   } if ((rating === "PG" || rating === "R") && (!isParentWith && age < 17)) {
    saleAllowed = false;
   }
   
   return saleAllowed;
```

Reply:
this code change, in my opinion, makes it even easier to read. "I'm declaring a variable here, then I'm executing my logic here, and then last I'm returning my output here".